custom.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module '*.svg' {
+  import React = require('react');
+  export const ReactComponent: React.FC<React.SVGProps<SVGSVGElement>>;
+  const src: string;
+  export default src;
+}
```

Comment:
let's pull up `types.d.ts` from from `services/` and put these in there with a comment that these are necessary for Jest to handle `svgs`